SpatialStructure-4/readme.md

```diff
@@ -0,0 +1,165 @@
+### Intent
+
+This UT establishes a spatial structure for the level crossing storyline. The intention is to serve as a skeleton for upcoming unit tests for this storyline.
+
+It is based on the storylines [SLLC-DD](https://app.box.com/folder/119147119179?s=q1x0vz7yqq7otrlet7dm2dx4u44s8fks) (IFC Rail phase 2) and [bSFi_IR_SLRD_DD](https://app.box.com/folder/122373280942?s=x9q3q62tcc77hdqpdw0vjncj7bg3byay) (IFC Infra Deployment)
+
+### Prerequisites
+
+None
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+- Georeferencing
+  - `IfcProjectedCRS`
+  - `IfcMapConversion`
```

Comment:
I'd rather see this as a dependency to Georeferencing-1.

Reply:
I am not sure that I understand? A dependency in what way?

Reply:
A link to a unit test does not mean, that the same *content* is being built upon, but the same *concepts* are being used. It means, that in order to implement this unit test, it is advisable to first implement the other one.

Reply:
But should it then reference all georeferencing UT:s? I would then consider it a reference to the topic and not a specific unit test?

Reply:
I see your point. It is difficult to visualize it in the graph, though. Additionally: the other goereferencing UT bring in different concepts that you probably don't need in this UT. So I'd go with the "minimal" requirement needed.

Reply:
Perhaps the Project Setup 1 would be a correct "parent"?